Gordon360/Controllers/LostAndFoundController.cs

```diff
@@ -120,5 +135,24 @@ public ActionResult<MissingItemReportViewModel> GetMissingItem(int id)
                 return NotFound();
             }
         }
+        /// <summary>
+        /// Get a missing item report with given ID.
```

Comment:
This is still referring to getting a missing item report, this comment needs to be updated to be correct.